src/ontology/uberon-edit.obo

```diff
@@ -224189,6 +224189,60 @@ intersection_of: part_of UBERON:0005610 ! iliac vein
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-06-17T14:34:50Z xsd:dateTime
 
+[Term]
+id: UBERON:8480001
+name: capillary of brain
+def: "A capillary that is part of the brain." [https://orcid.org/0000-0002-0819-0473]
+intersection_of: UBERON:0001982 ! capillary
+intersection_of: part_of UBERON:0000955 ! brain
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-06-24T09:28:31Z xsd:dateTime
+
+[Term]
+id: UBERON:8480002
+name: thoracic aorta smooth muscle tissue
+def: "A portion of smooth muscle tissue that is part of the thoracic aorta." [https://orcid.org/0000-0002-0819-0473]
+intersection_of: UBERON:0004178 ! aorta smooth muscle tissue
+intersection_of: part_of UBERON:0001515 ! thoracic aorta
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-06-24T09:30:54Z xsd:dateTime
+
+[Term]
+id: UBERON:8480003
+name: iliac artery smooth muscle tissue
+def: "A portion of smooth muscle tissue that is part of an iliac artery." [https://orcid.org/0000-0002-0819-0473]
+intersection_of: UBERON:0002111 ! artery smooth muscle tissue
+intersection_of: part_of UBERON:0005609 ! iliac artery
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-06-24T09:32:07Z xsd:dateTime
+
+[Term]
+id: UBERON:8480004
+name: iliac vein endothelium
+def: "An endothelium that is part of an iliac vein." [https://orcid.org/0000-0002-0819-0473]
+intersection_of: UBERON:0001919 ! endothelium of vein
+intersection_of: part_of UBERON:0005610 ! iliac vein
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-06-24T09:33:18Z xsd:dateTime
+
+[Term]
+id: UBERON:8480005
+name: placental artery endothelium
+def: "An endothelium that is part of the placental artery." [https://orcid.org/0000-0002-0819-0473]
+intersection_of: UBERON:0001917 ! endothelium of artery
+intersection_of: part_of UBERON:0022358 ! placenta blood vessel
```

Comment:
I wonder if it's worth adding `placental artery` (and I guess vein by that logic). That would give it the same pattern as your other terms which would be useful if we decide to implement patterns in uberon.

Reply:
OK. I will create placental artery and placental vein.
@shawntanzk should I keep placental artery endothelium child of placenta blood vessel or make it child of placental artery?

Reply:
it should follow the pattern of all the other terms, so: 
equivalent: endothelium and ("part of" some "placental artery")